vue/components/organisms/personalization/personalization.vue

```diff
@@ -242,6 +242,7 @@ export const Personalization = {
         this.$bus.bind("pre_config", () => {
             this.enabled = false;
             this.form = null;
+            this.valid = true;

```

Comment:
What happens if:
1 - I make an invalid personalization
2 - I change the model to some model that uses the same personalization plugin

?

Is the state of the personalization form reset?

Reply:
Yes, it resets. That scenario is working as expected in master.

As the bug only happens when we change from one personalization plugin that has computed `valid` to one that doesn't, if we make an invalid personalization in a plugin that has a computed `valid` than change the model to some model that uses the same personalization plugin, we're changing to a plugin that uses computed `valid`.
This bug only happens when we switch from plugin that uses computed `valid` to one that doesn't.

Here's a GIF of it working in ripe-white-now
![Peek 2020-12-02 09-22](https://user-images.githubusercontent.com/24736423/100853664-ee64b380-347f-11eb-90b1-727727d34af8.gif)

Reply:
I know that empirically that happens, the question is: who is calling `reset`? Because if it's the plugin that it is resetting itself maybe there's some fixing to do since it should be `personalization` doing it.

Reply:
I checked and it's computed that is calling event `update:reset` with value set to true
This is because in reference the computed `valid` is set with [immediate to true](https://github.com/ripe-tech/ripe-commons-pluginus/blob/master/vue/components/forms/personalization-form/reference.vue#L173)

![image](https://user-images.githubusercontent.com/24736423/100857394-8ebcd700-3484-11eb-8545-d77c0233f8b5.png)